hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -1,11 +1,73 @@
 module HookContextExtension
+  # FIXME: remove when #23332 is released
+  def param_value(mod, name)
+    param(mod, name).value if param(mod, name)
+  end
+
   def success_file
     File.join(File.dirname(kafo.config.config_file), '.installed')
   end
 
   def new_install?
     !File.exist?(success_file)
   end
+
+  def ensure_package_installed(package)
+    bin_path = Kafo::PuppetCommand.search_puppet_path('puppet')
```

Comment:
I think I can call this this way from the hooks.

Reply:
That was going to be my suggestion.